gravitee-apim-console-webui/src/organization/configuration/user/detail/org-settings-user-detail-add-group-dialog.component.html

```diff
@@ -54,6 +54,19 @@ <h2 mat-dialog-title>Add a group with roles</h2>
           >
         </mat-select>
       </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>Integration Role</mat-label>
+        <mat-select formControlName="integrationRole" aria-label="Integration role">
+          <mat-option>-- None --</mat-option>
+          <mat-option
+            *ngFor="let integrationRole of integrationRoles$ | async"
+            [disabled]="integrationRole.system"
+            [value]="integrationRole.name"
+            >{{ integrationRole.name }}</mat-option
+          >
```

Comment:
Should we create new ngFor or use only new @For?

Do we have rule about this @pes-spyro-soft-com?

Reply:
I am not strict about @for syntax. There are still cases where the *ngFor directive can not be replaced by @for. 

IMHO: Iterating over observable wiht @for would look bit ugly to me :-) 
```
@for (integrationRole of (integrationRoles$ | async); track integrationRole.id) {
    {{ integrationRole.name }}
}
```